README.md

```diff
@@ -22,31 +25,36 @@ installed by running this command in the terminal.
 
 `sudo apt install virtualenv git python3-dev gcc`
 
+If you are installing onto Windows, you will need to have installed Visual Studio Build Tools, and a 64-bit version of
+Python.
+
 Clone the Onko repository
 
 `git clone https://github.com/didymo/OnkoDICOM.git`
 
 Enter the directory and create a virtual environment with a name of
-your choice, in this case it's envOnkoDICOM.
+your choice, in this case it's venv.
 
 `cd OnkoDICOM`
 
 `virtualenv --python=python3 venv`
 
+Note that when cloning into a PyCharm workspace, it is recommended to create a virtual environment from the terminal
+_outside_ of PyCharm, as PyCharm's built-in virtual environment creation often leads to issues with the pip version.
+
 Activate the virtual environment
 
 `source venv/bin/activate`
 
-OR
+OR for Windows
 
 `venv/Scripts/activate.bat`
 
 Install the requirements
 
-`pip install -r requirements.txt`
-
-`pip install --no-deps -r requirements-without-deps.txt`
+`pip install -r pre-requirements.txt`
```

Comment:
what version of pip were you using?
while I have seen some weird things with pip, this seems like there might be something going on that should either be in a bug report or a release note in those other projects

Reply:
The base version of pip that is created with the virtual environment is adequate. The issue relating to the installed pip version is related to the way that specifically PyCharm installs pip into the virtual environment, but if the environment is created from the command line outside of PyCharm, it works correctly. This is a widely experienced issue with PyCharm apparently, e.g. https://github.com/pypa/pip/issues/5343